apps/prairielearn/elements/pl-drawing/elements.py

```diff
@@ -980,7 +980,7 @@ class ArcVector(BaseElement):
     def generate(el, data):
         disregard_sense = pl.get_boolean_attrib(el, "disregard-sense", False)
         color = pl.get_color_attrib(el, "color", "purple")
-        clockwise_direction = pl.get_boolean_attrib(el, "clockwise-direction", True)
+        clockwise_direction = pl.get_boolean_attrib(el, "clockwiseDirection", True)
```

Comment:
If I'm not mistaken, currently it is possible to use either `type="pl-arc-vector-CW"` or `type="pl-arc-vector" clockwise-direction="true"` with similar effects. Make sure that this remains, for backwards compatibility, as you don't want to break user's questions that use either option. In other words, the attribute for the individual elements should remain `clockwise-direction`, the only thing that changes is the option used on the JS side.

Reply:
That makes sense